docs/operator/test.md

```diff
@@ -0,0 +1,94 @@
+# Test guide
+
+---
+This document facilitates an example of working commands to run two networks on a set of drand daemons. Nodes will be created using `docker-compose`.
+The file can be found on drand repo, under `<project-folder>/test/docker*utils/docker-compose.yml`.
+
+## Pre requisites
+
+---
+1) Build docker images as follows:
+
+```bash
+make build_docker_all
```

Comment:
This doc is great ! Can we just put explicitely all the native commands ? For this one for example, that means the operator must go to the github and look the right folder and look the Makefile and then the docker compose file... Let's just give the native commands here and save him a bunch of time :)

Reply:
I am not sure what we will get from putting the docker command directly there, as the operator will need the GitHub repo in order to build the docker image. Therefore, he will have access to the makefile and the `make build_docker` command. However, I have put the native command on the doc!